README.md

```diff
@@ -57,5 +82,239 @@ Due to the 32 characters limitation, it can be truncated.
 When the controller detects that a managed load balancer for the current cluster doesn't have a matching ingress 
 resource anymore, it deletes all the previously created resources.
 
+## Building
+
+This project provides a `Makefile` that can be used to build a binary or to build a docker image for it. To build a binary for your current operating system, it is enough to run `make` or `make build.linux` to build an image for Linux. 
+To create a docker image, you can execute `make build.docker` instead. 
+
+## Deploy
+
```

Comment:
Describing skipper as part of the overall solution and not mate seems incomplete. We do mention hostnames.